force-app/main/selector/ContactSelector.cls

```diff
@@ -62,6 +62,20 @@ public inherited sharing class ContactSelector {
         return Database.Query(householdMembersQueryFor(queryHHIds));
     }
 
+     public Contact getDonationsSummaryForContact(Id contactId) {
+        if (!Schema.SObjectType.Contact.isAccessible()) {
+            throw new NoAccessException();
+        }
+
+        return [SELECT ID,
+                       npo02__TotalOppAmount__c,
+                       npo02__OppAmountThisYear__c,
+                       npo02__OppAmountLastYear__c 
+                FROM   Contact
+                WHERE  ID = :contactId WITH SECURITY_ENFORCED 
```

Comment:
Andrew mentioned it above, but we'll want to catch, or separately check, the scenario where a User doesn't have permissions to these fields and display an error message.

Reply:
agreed